### PR TITLE
Add org.freedesktop.Sdk.Extension.gnat14

### PR DESCRIPTION
**Pull request by jyhi**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly. **Freedesktop SDK Extension - GNAT 14**
- [X] The domain used for the application ID is [controlled by the application developer(s)][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2].
- [X] I have [built][build] and tested the submission locally.
- [X] ~~I am an author/developer/upstream contributor to the project. If not,~~ I contacted upstream developers about this submission. **Link: <https://alire.ada.dev/>**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "only-arches": ["x86_64"],
```
Comment (hfiguiere) [inline flathub.json:2]: why not aarch64?

Comment (hfiguiere) [reply flathub.json:2]: so building from source still not tried? that's very disappointing.

File: org.freedesktop.Sdk.Extension.gnat14.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      <li>Alire (alr) - the Ada Library Repository tool</li>
+    </ul>
+  </description>
+  <categories>
+    <category>Development</category>
+  </categories>
+  <developer id="dev.ada.alire">
+    <name>Ada Library Repository</name>
+  </developer>
+  <content_rating type="oars-1.0" />
```
Comment (bbhtt) [inline org.freedesktop.Sdk.Extension.gnat14.metainfo.xml:23]:
```suggestion
  <content_rating type="oars-1.1" />
```

File: flathub.json (hunk 1 above)
Comment (jyhi) [reply flathub.json:2]: I'd say I personally don't have this spare time, so apart from any upstream effort I can only do what I can do.

File: org.freedesktop.Sdk.Extension.gnat14.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    build-commands:
+      - install -Dm755 -t /usr/lib/sdk/gnat14 enable.sh
+    sources:
+      - type: file
+        path: enable.sh
+
+  - name: metadata
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 -t /usr/lib/sdk/gnat14/share/metainfo org.freedesktop.Sdk.Extension.gnat14.metainfo.xml
```
Comment (bbhtt) [inline org.freedesktop.Sdk.Extension.gnat14.yml:67]:
```suggestion
      - install -Dm644 -t ${FLATPAK_DEST}/share/metainfo org.freedesktop.Sdk.Extension.gnat14.metainfo.xml
```

File: org.freedesktop.Sdk.Extension.gnat14.metainfo.xml (hunk 2 above)
Comment (jyhi) [reply org.freedesktop.Sdk.Extension.gnat14.metainfo.xml:23]: <https://hughsie.github.io/oars/generate.html> says 1.1 works with GNOME Software >= 3.27.3. Won't a 1.0 markup here provide more compatibility (provided that this is an empty tag)?

Comment (bbhtt) [reply org.freedesktop.Sdk.Extension.gnat14.metainfo.xml:23]: No, that website hasn't been updated it years and the version is ancient.

Comment (jyhi) [reply org.freedesktop.Sdk.Extension.gnat14.metainfo.xml]: Changed.

**Comment by bbhtt** [comment]
bot, build org.freedesktop.Sdk.Extension.gnat14

**Pull request closed without merging**